script.js

```diff
@@ -434,60 +450,108 @@ function placeCard(event) {
+      if (placedCard[0] > upCardValues[3]) {
+        console.log('you win');
+        cell.dataset.owner = owner ;
+      } else if (placedCard[0] == upCardValues[3]) {
+        console.log('draw');
+      } else if (placedCard[0] < upCardValues[3]) {
+        console.log('you lose');
+        dest.dataset.owner  = cell.dataset.owner;
+      }
+    }
+    if (cell.dataset.cell == left && cell.innerText !== '' && destCell !== 1 && destCell !== 4 && destCell !== 7) {
+      let leftCardValues = [...cellArray[(destCell - 2)].innerText].filter(item => item !== '\n');
+      console.log('left battle', placedCard[1], 'vs.', leftCardValues[2]);
+      if (placedCard[1] > leftCardValues[2]) {
+        console.log('you win');
+        cell.dataset.owner = owner ;
+      } else if (placedCard[1] == leftCardValues[2]) {
+        console.log('draw');
+      } else if (placedCard[1] < leftCardValues[2]) {
+        console.log('you lose');
+        dest.dataset.owner  = cell.dataset.owner;
+        console.log(owner)
+      }
+    }
+    if (cell.dataset.cell == right && cell.innerText !== '' && destCell !== 3 && destCell !== 6 && destCell !== 9) {
+      let rightCardValues = [...cellArray[destCell].innerText].filter(item => item !== '\n');
+      console.log('right battle', placedCard[2], 'vs.', rightCardValues[1]);
+      if (placedCard[2] > rightCardValues[1]) {
+        console.log('you win');
+        cell.dataset.owner = owner ;
+      } else if (placedCard[2] == rightCardValues[1]) {
+        console.log('draw');
+      } else if (placedCard[2] < rightCardValues[1]) {
+        console.log('you lose');
+        dest.dataset.owner  = cell.dataset.owner;
+        console.log(owner)
+      }
+    }
+    if (cell.dataset.cell == down && cell.innerText !== '') {
+      let downCardValues = [...cellArray[(destCell + 2)].innerText].filter(item => item !== '\n');
+      console.log('down battle', placedCard[3], 'vs.', downCardValues[0]);
+      if (placedCard[3] > downCardValues[0]) {
+        console.log('you win');
+        cell.dataset.owner = owner;
+      } else if (placedCard[3] == downCardValues[0]) {
+        console.log('draw');
+      } else if (placedCard[3] < downCardValues[0]) {
+        console.log('you lose');
+        dest.dataset.owner  = cell.dataset.owner;
+      }
+    }
+  }
+}
 
-// // winner card select
-// function winnerCardSelect(event) {
-//     console.log("Winner Card Selected");
+// // check for game winner
+// function checkForWin() {
 // }
 
-// // reset table per round
-// function resetTableRound(table) {
-//     console.log("Resetting For Next Round...");
-// }
 
-// // randomize winner keeper card
-// function randomKeeper(table) {
-//     console.log("Randomizing Keeper...");
+// start game
+// function startGame() {
 // }
 
-// // reset table per game
-// function resetTableGame(table) {
-//     console.log("Resetting For Next Game...");
-// }
+// // new game
+// function newGame() {
+// }
```

Comment:
```suggestion
// }

```
Don't forget the new line :)

Reply:
Could you provide a little more info as to why that is?

Reply:
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

I thought it was more of a Python thing, but GitHub wants a new line in JS as well

Reply:
Got it, thank you!

One of those it obviously can't hurt, but figured there was a why.

Also feel free to tell me to google it myself next time haha.